src/Translation/German/translation.json

```diff
@@ -353,21 +350,19 @@
   "signIn": {
     "back": "Zurück",
     "signUp": "Registrieren",
-    "existingUserTitle": "Bei Portato anmelden",
-    "existingUserText": "Melde dich mit deiner E-Mail-Adresse und deinem Passwort bei Portato an.",
-    "nonExistingUserTitle": "Noch kein Profil?",
+    "existingUserTitle": "Anmelden",
+    "nonExistingUserTitle": "Du hast noch kein Profil?",
     "nonExistingUserText": "Erstelle in nur wenigen Schritten ein Profil und werde Teil der Portato-Gemeinschaft.",
```

Comment:
Same here: Maybe better to keep it short: "Erstelle ein Profil und werde Teil der Portato-Gemeinschaft."